app/containers/Questions/saga.js

```diff
@@ -1,24 +1,71 @@
 import { takeLatest, call, put, select } from 'redux-saga/effects';
 
 import { selectEos } from 'containers/EosioProvider/selectors';
-import { getQuestions } from 'utils/questionsManagement';
+import {
+  getQuestions,
+  getQuestionsFilteredByCommunities,
+} from 'utils/questionsManagement';
 
-import { GET_QUESTIONS_LIST } from './constants';
-import { getQuestionsListSuccess, getQuestionsListError } from './actions';
+import { GET_INIT_QUESTIONS, GET_NEXT_QUESTIONS } from './constants';
 
-export function* getQuestionsListWorker(res) {
+import {
+  getInitQuestionsSuccess,
+  getInitQuestionsError,
+  getNextQuestionsSuccess,
+  getNextQuestionsError,
+} from './actions';
+
+export function* getInitQuestionsWorker({ limit, offset, communityIdFilter }) {
+  try {
+    const eosService = yield select(selectEos);
+
+    let questionsList = [];
+
+    if (communityIdFilter) {
+      questionsList = yield call(() =>
+        getQuestionsFilteredByCommunities(
+          eosService,
+          limit,
+          offset,
+          communityIdFilter,
+        ),
+      );
+    } else {
+      questionsList = yield call(() => getQuestions(eosService, limit, offset));
```

Comment:
This part I think is a bit more complex. If the community is not selected then we need to display all questions from communities **followed / selected** by user.
Was the logic for the user to follow communities implemented on the backend?

Reply:
this logic will be realized soon